src/Twilio/Clients/TwilioRestClient.cs

```diff
@@ -92,6 +107,11 @@ public Response Request(Request request)
             try
             {
                 response = HttpClient.MakeRequest(request);
+                if(LogLevel == "debug")
+                {
+                    Console.WriteLine("response.status: " + response.StatusCode);
+                    Console.WriteLine("response.headers: " + "{"+ response.Headers + "}");
```

Comment:
Why is this one specifically wrapped in braces?

Reply:
Was originally for readability because each header is logged onto a new line but the Node logging has the same format with just a label so I will remove the braces.